packages/utils/logger/src/formats/exclude-colors.ts

```diff
@@ -1,17 +1,13 @@
-'use strict';
-
-const { format } = require('winston');
+import { format } from 'winston';
 
 /**
  * This will remove the chalk color codes from the message provided.
  * It's used to log plain text in the log file
  */
-const excludeColors = format.printf(({ message }) => {
+export default format.printf(({ message }: { message: string }) => {
```

Comment:
What do you think of upgrade the excepted type to ensure we still have access to other properties?
Maybe something like
```ts
import { format, Logform } from 'winston';

interface TransformableInfo extends Logform.TransformableInfo {
  message: string;
}

/**
 * This will remove the chalk color codes from the message provided.
 * It's used to log plain text in the log file
 */
export default format.printf(({ message }: TransformableInfo) => {
  return message.replace(
    // eslint-disable-next-line no-control-regex
    /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g,
    ''
  );
});

```

Reply:
I'll probably just use the winston interface in that case as it's not something we could ever extend anyway